Tests/Shared/Core/UnitTestBase.swift

```diff
@@ -165,13 +165,7 @@ open class UnitTestBase<Component>: XCTestCase {
     }
 
     public func runOnBackground(_ block: @escaping () -> Void) {
-        CioThreadUtil().runBackground {
-            block()
-        }
-    }
-
-    public func runOnMain(_ block: @escaping () -> Void) {
```

Comment:
Removed `runOnMain` since there is no more purpose behind it. The reason it was added was to make sure that a test function could create 2+ different threads to run code on. However, the existing function `runOnBackground` already creates a new thread every time that the function is called. Therefore, no reason for `runOnMain`.